generatebundlefile/main.go

```diff
@@ -138,11 +142,31 @@ func cmdPromote(opts *Options) error {
 		},
 	}
 
+	Profile := "default"
+	val, ok := os.LookupEnv("AWS_PROFILE")
+	if ok {
+		Profile = val
+	}
+	if opts.regionalBuildMode && Profile != "default" {
```

Comment:
If we are running the promote in regional mode and profile is staging/prod, we need a public ECR release clients (corresponding to staging or prod accounts), else the default dev clients will suffice.